### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by yugasun**
**What type of PR is this?**

更新 README

**What this PR does / why we need it**:

中文版 README 翻译不够彻底

**Which issue(s) this PR fixes**:

如 commit 所示

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
-- [Documents](./docs/index.md)
+- [安装](./docs/getting_started.md)
+- [文档](./docs/index.md)
 
-## Key Features
+## 核心功能
 
 > 提示：火币公链目前正处在早期开发中，技术细节、设计文档和实现代码会频繁变更。
 
-### Parallel Execution Consensus
```
Comment (zhouyun-zoe) [inline README.md]: “Parallel” is supposed to be translated in here.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
+### 共识算法
 
 火币公链采用 [Overlord][overlord] 共识算法，其设计目标是成为能够支持上百个共识节点，满足数千笔每秒的交易处理能力，且交易延迟不超过数秒的 BFT 共识算法。Overlord 的核心思想是解耦交易定序与状态共识，从而实现共识和执行完全并行，极大提高整条链的交易吞吐量。
 
-### RISC-V Based Virtual Machine
```
Comment (zhouyun-zoe) [inline README.md]: "Virtual Machine" should be translated to "虚拟机“

File: README.md [hunk 3]
```diff
@@ -1,43 +1,44 @@
-# Huobi Chain
```
Comment (zhouyun-zoe) [inline README.md]: there is no need to translate 'Huobi Chain‘

**Comment by yugasun** [comment]
@zhouyun-zoe All suggestions had been updated.

**Pull request merged**